spring-integration-core/src/main/java/org/springframework/integration/handler/AbstractReplyProducingMessageHandler.java

```diff
@@ -147,7 +147,19 @@ protected Object doInvokeAdvisedRequestHandler(Message<?> message) {
 
 	}
 
-	private class AdvisedRequestHandler implements RequestHandler {
+	/**
+	 * Used to wrap the
+	 * {@link AbstractReplyProducingMessageHandler#handleRequestMessage(Message)}
+	 * method. Also allows access to the underlying
+	 * {@link AbstractReplyProducingMessageHandler} to obtain properties.
+	 *
+	 * @author Gary Russell
+	 * @since 4.3.2
```

Comment:
Oh! Sorry. I saw two versions in JIRA, but didn't notice that another one is `5.0 M1`.
Since we haven't switched there yet, I don't see reason to worry about `5.0`.
All changes to `4.3.x` will be there anyway.